common/common.go

```diff
@@ -137,6 +138,19 @@ func SupportsOverlay() bool {
 	return false
 }
 
+// GetAppByImageName retrieves an app from the AppList whose image has the
+// specified name; if there is no such app, nil is returned. The returned
+// *RuntimeApp MUST be considered read-only.
+func GetAppByImageName(al schema.AppList, name types.ACIdentifier) *schema.RuntimeApp {
```

Comment:
In the commit log, you have:
```
common: add GetAppByImageName function
These should eventually be moved to appc/spec but they live in
rkt/common for now so we can release rkt with appc/spec v0.6.0.
```

Is it still exact?

If so, you could add a TODO and a reference to the issue/PR in appc/spec if it exists.